### PR TITLE
[PFC WD]: Initial support

### PR DESCRIPTION
**Pull request by marian-pritsak**
Changes include support fur dynamic subscription to syncd counters, needed for PFC WD.
libsairedis was extended to support queue counters serialization.

### CODE REVIEW

**Comment by qiluo-msft** [comment]
Retest this please

**Comment by qiluo-msft** [comment]
Retest this please.

File: syncd/syncd.cpp [hunk 1]
```diff
@@ -2309,6 +2310,62 @@ sai_status_t processEvent(
...
+
+        if (op == DEL_COMMAND)
+        {
+            PfcWatchdog::removeQueue(queueVid);
+            continue;
+        }
+
+        auto idStrings  = swss::tokenize(value, ',');
+
+        if (field == PFC_WD_PORT_COUNTER_ID_LIST)
```
Comment (qiluo-msft) [inline syncd/syncd.cpp]: not declared

Comment (marian-pritsak) [reply syncd/syncd.cpp]: https://github.com/Azure/sonic-swss-common/pull/124 should be merged first

**Comment by marian-pritsak** [comment]
retest this please

**Comment by marian-pritsak** [comment]
retest this please

**Comment by marian-pritsak** [comment]
@stcheng @lguohan No more conflicts/dependencies. Build passes.

**Pull request merged**